### DISCUSSION TITLE
图片无法自动上传

### DISCUSSION THREAD

**Discussion by plantegg** [drmingdrmer/md2zhihu#11] [Q&A]
图片还是生成在本地，知乎导入生成后的markdown时无法自动上传图片(我的姿势不对?），看示例应该是自动帮我上传到某个图床才对啊

取图片路径的时候 src = src[1:] 自动把第一个字符丢了，改成 src = src[0:] 就可以了
![image](https://user-images.githubusercontent.com/478304/125605610-7441fd5a-029e-4afe-99f3-d21bf2fe9b0c.png)

**Comment by drmingdrmer** [comment]
我看看你的源文件 和 你build出来的文件? 看起来你在md里用了绝对路径?
